### ISSUE TITLE
[BUG] [ws-sbom-generator] Latest docker image exits before executing the tool when run from command line

### ISSUE THREAD

**Issue by olhado** [whitesource-ps/ws-sbom-generator#143]
**Bug Description**  
Latest docker image (whitesourcetools/ws-sbom-generator:22.9.1) doesn't generate an SBOM because the container exits the run before the HEALTHCHECK can execute. The container run exits without error.

**Steps to Reproduce**  
1. Run `docker pull whitesourcetools/ws-sbom-generator:latest`
2. Run the following command:
     ```
     docker run --name ws-sbom=generator \
         -v $EXTRA_CONF_DIR:/opt/ws-sbom-generator/sbom_generator/resources \
         -v $REPORT_OUTPUT_DIR:/opt/ws-sbom-generator/sbom_generator/output \
         -e WS_USER_KEY=<USER_KEY> \
         -e WS_TOKEN=<WS_ORG_KEY> \
         -e WS_SCOPE_TOKEN=<WS_PROJECT_TOKEN> \
         -e WS_URL=<COMPANY_WS_URL> \
         -e WS_REPORT_TYPE=json \
         whitesourcetools/ws-sbom-generator
    ```
5. The command returns with no output to stdout or the $REPORT_OUTPUT_DIR

**Expected Behavior**  
I expect an SBOM JSON file to be output to my $REPORT_OUTPUT_DIR

**Environment Details**  
 - OS: macOS Monterey (12.6.1)
 - CPU: Apple M1 Pro
 - ws_sbom_generator 22.9.1
 
**Additional Context**  
Running the command after installing the python package for 22.9.1 works, and running the container image `whitesourcetools/ws-sbom-generator:0.6.8` works, both on the same system. I would like to use the latest docker container in my CI/CD pipeline.

My volumes are mounted with full paths, not relative paths.

**Comment by olhado** [comment]
Any thoughts? Are the newer images working as designed?

**Comment by DimarrWS** [comment]
Hi @olhado ! I will check the issue. 
Are you running it for the same project you sent me by email?

**Comment by olhado** [comment]
Yup. to be clear, 0.6.8 works with `docker run`, but 22.9.1 (haven't tried latest) exits before any work is done.

**Comment by olhado** [comment]
This appears to be fixed in the latest 22.12.2.1!
